Plugins/ServerPlugin.cpp

```diff
@@ -266,7 +266,7 @@ ServerPlugin::ServerPlugin(MainWindow& mainWindow) : RGMPlugin(mainWindow) {
     qDebug().noquote() << "Environmental variable \"MSYS_ROOT\" is not set defaulting MSYS path to: " + msysPath;
   } else msysPath = env.value("MSYS_ROOT");
 
-  msys2Proc->start(msysPath + "/msys2_shell.cmd");
+  msys2Proc->start(msysPath + "/msys2_shell.cmd -mingw64");

```

Comment:
This did not fix the missing `draw_*` completions. However, I looked some more and noticed that all of the `graphics_mandatory.h` functions are available, such as `screen_refresh` and the like. All of the window functions are also available, leading me to believe we just have the graphics system set to `None` or something.

Reply:
did it fix all those incude errrs for map n etc in jdi log?

Reply:
I don't know, all I know is I compared the log with regular normal emake that I know works and the one from RGM is way longer. I notice a bunch of stuff about var4 but I did not see anything about map in there.

Reply:
https://pastebin.com/Fjw9gshc like line 73 in your earlier paste. are you still seeing those errors?

Reply:
Two files are identical according to online diff viewer.

Reply:
im fairly certain thats the issue. i doubt youll see those errors launching emake normally

Reply:
Try 638b4619aa67a48e4b19527f62879bc3dd615b4d

Reply:
Still nothing, are you sure it's not just the JDI STL hacks to fix string or something?

Reply:
```
$ ./enigma-dev/emake --list --enigma-root C:/Users/Owner/Documents/projects/enigma-dev
LibEGM parsing log at: C:\msys64\tmp\/enigma_libegm.log
ENIGMA compiler log at: C:\msys64\tmp\/enigma_compiler.log
make: *** No rule to make target 'required-directories'.  Stop.
Segmentation fault
```

Reply:
I did 1c0c3e2ab07de5d05a0f9984cb404490195a8464 which just printed empty. So that means the environment is not copying anything. As I said, I believe the process is not having enough time to actually initialize.